### PR TITLE
Pokemon api files

### PR DESCRIPTION
**Pull request by satchelsmidt**
comment

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,50 +1,40 @@
-$(document).on("click", function(event) {
+//Initialize function when user clicks anywhere on the document
+$(document).on("click", function() {
 
     // Here we grab the text from the input box
     // var pokemon = $("#pokeMon").val();
```
Comment (jennhenn18) [inline app.js:5]: Why is this commented out?

File: app.js [hunk 2]
```diff
@@ -1,50 +1,40 @@
...
-    // Write code between the dashes below to hit the queryURL with $ajax, then take the response data
-    // and display it in the div with an id of movie-view
+    
+var count = 1;
+
+function fetchPokemon(){
+    
+    // Here we construct our URL, and pick our pokemon randomly by its 'id number' (pokedex number). In this case, we limit our results to the first 150 pokemon (OG)
+    
+    // var queryURL = "https://pokeapi.co/api/v2/pokemon/"+ Math.floor(Math.random()*150)+"/"
```
Comment (jennhenn18) [inline app.js:13]: should this comment be removed? Not sure what is it for

File: app.js [hunk 3]
```diff
@@ -1,50 +1,40 @@
...
+    var queryURL = "https://pokeapi.co/api/v2/pokemon/"+count+"/"
```
Comment (jennhenn18) [inline app.js:14]: what is driving this +count+?

**Review by jennhenn18 — CHANGES_REQUESTED** [review]
Overall feedback:

Please add a task to the Project MVP backlog board to disable the submit button using JavaScript
With var count = 1 inside the onclick function we need to disable the button so users cannot click the button while the API call is running.

File: app.js (hunk 1 above)
Comment (satchelsmidt) [reply app.js:5]: Initial testing process that was not yet removed, will be removed in next push!

File: app.js (hunk 2 above)
Comment (satchelsmidt) [reply app.js:13]: Initial formatting of queryURL, leaving for review purposes until final API call is devised

File: app.js (hunk 3 above)
Comment (satchelsmidt) [reply app.js:14]: var count, defined outside of function (discussed already)

**Review by jennhenn18 — APPROVED** [review]

**Pull request merged**